myEnigma/myEnigma.ino

```diff
@@ -1461,9 +1461,9 @@ void setConfig(enigmaModel_t model,
 	       uint8_t rotor2,
 	       uint8_t rotor3,
 	       uint8_t etw,
-	       char    ringSetting[WALZECNT],	//=" AAA"
-	       char    plugboard[40],		//="AB CD EF GH IJ KL MN OP QR ST UV WX YZ",
-	       char    currentrotor[WALZECNT]	//=" AAA"
+	       const char    ringSetting[WALZECNT],	//=" AAA"
+	       char          plugboard[40],		//="AB CD EF GH IJ KL MN OP QR ST UV WX YZ",
+	       const char    currentrotor[WALZECNT]	//=" AAA"
```

Comment:
Closer look and in no case is it actually changed, it's just a parameter passed in so it looks ok.